<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExtract2</class>
 <widget class="QWidget" name="FileExtract2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>639</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>FileExtract2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>main.ico</normaloff>main.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>搜索目录：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_fuzzy">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>模糊搜索</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_recursion">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>递归搜索</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_searchFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>搜索文件</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_searchDir">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>搜索目录</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="6,1,1">
         <item>
          <widget class="QLineEdit" name="lineEdit_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_scaninput">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_search">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>文件列表：（可编辑）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_scanlist">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>搜索结果：（可删除）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listView">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="gridSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_delete">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_searchresult">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>搜索结果</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>提取目录：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_override">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>允许覆盖</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="6,1,1">
         <item>
          <widget class="QLineEdit" name="lineEdit_output">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_scanoutput">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_extract">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>提取</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>textEdit</sender>
   <signal>textChanged()</signal>
   <receiver>FileExtract2</receiver>
   <slot>textEidt_textChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_search</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>searchClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_delete</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>deleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_extract</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>extractClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_scaninput</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>scanInputClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_scanoutput</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>scanOutputClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_scanlist</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>scanListClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_searchresult</sender>
   <signal>clicked()</signal>
   <receiver>FileExtract2</receiver>
   <slot>searchResultClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>textEidt_textChanged()</slot>
  <slot>searchClicked()</slot>
  <slot>deleteClicked()</slot>
  <slot>extractClicked()</slot>
  <slot>scanInputClicked()</slot>
  <slot>scanOutputClicked()</slot>
  <slot>scanListClicked()</slot>
  <slot>searchResultClicked()</slot>
 </slots>
</ui>
